src/ui/components/UserReview/styles.scss

```diff
@@ -36,4 +36,19 @@
   color: $sub-text-color;
   display: flex;
   font-size: $font-size-s;
+
+  a,
+  a:link,
+  a:visited {
+    color: $grey-50;
```

Comment:
Ah, shoot. This needs to be `$sub-text-color` to match the text. They are actually different. Oops.